aiida/tools/importexport/dbexport/utils.py

```diff
@@ -385,12 +385,11 @@ def retrieve_linked_nodes(process_nodes, data_nodes, **kwargs):  # pylint: disab
 
     :raises `~aiida.tools.importexport.common.exceptions.ExportValidationError`: if wrong or too many kwargs are given.
     """
-    from aiida.common.links import LinkType, GraphTraversalRules
-    from aiida.orm import Data
+    from aiida.common.links import GraphTraversalRules
```

Comment:
This entire function can be replaced by a simple `traverse_graph(..., graph_traversal_rules=GraphTraversalRules.EXPORT)` once that is implemented